lib/testing/src/lib/protractor/core/pages/material/date-picker-calendar.page.ts

```diff
@@ -80,4 +81,30 @@ export class DatePickerCalendarPage {
         await BrowserActions.click(endDayElement);
         await this.checkDatePickerIsNotDisplayed();
     }
+
+    private async setDateUsingPeriodButton(date: Date) {
+        const months = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC'];
+        const year = date.getFullYear(), month = months[date.getMonth()], day = date.getDate();
```

Comment:
can you declare each variable on it's own line? Just for readability so the eye can catch easier the declared variables and not miss any.